supabase/seed.sql

```diff
@@ -182,12 +182,12 @@ INSERT INTO "auth"."refresh_tokens" ("instance_id", "id", "token", "user_id", "r
 -- Data for Name: conversations; Type: TABLE DATA; Schema: public; Owner: postgres
 --
 
-INSERT INTO "public"."conversations" ("id", "created_at", "member_has_deleted") VALUES
-	(1, '2024-06-16 13:44:29.042856+00', false),
-	(2, '2024-06-16 13:46:10.25272+00', false),
-	(3, '2024-06-16 13:47:48.353521+00', false),
-	(4, '2024-06-16 13:49:04.157526+00', false),
-	(5, '2024-06-16 13:51:15.83193+00', false);
+INSERT INTO "public"."conversations" ("id", "created_at") VALUES
+	(1, '2024-06-16 13:44:29.042856+00'),
+	(2, '2024-06-16 13:46:10.25272+00'),
+	(3, '2024-06-16 13:47:48.353521+00'),
+	(4, '2024-06-16 13:49:04.157526+00'),
+	(5, '2024-06-16 13:51:15.83193+00');
```

Comment:
I might be acting silly but are we not missing this property now? My understanding is `member_has_deleted` was moved to `user_conversations` but I can't see it in the data

Reply:
oh this is a good catch I dont think I want to mess with the seed anymore now I think just removing it from the request is the way to go since the partner has deleted defaults to false and we start with no deleted conversations in the tests

Reply:
Just want to make sure we've got what you intended. Right now the changes to seed are in the pr but when I run dev I can see `partner_has_deleted` is in `user_conversations`, not `conversations`. I think this is what we wanted from the start, yes ?

The tests are looking great, by the way 👌

Reply:
yeah it is supposed to be there